src/Tribe/Views/V2/functions/classes.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * Calendar Class Functions
+ *
+ * @since TBD
+ */
+namespace Tribe\Events\Views\V2;
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: f67a7e66f289fe4887f8967644f3c049</sub>